src/main/java/org/apache/sling/caconfig/ConfigurationBuilder.java

```diff
@@ -86,4 +86,15 @@
      */
     @NotNull <T> Collection<T> asAdaptableCollection(@NotNull Class<T> clazz);
 
+    /**
+     * Check if the configuration node is existing in the JCR. This method checks for the configuration existence based on {@code configName} defined in the
```

Comment:
nitpicking: JCR? Or rather "resource hierarchy"? It's often used as synonym (as hierarchies are often quite identical), but let's be clear here :-)

Reply:
thanks, I have updated the comments